### DISCUSSION TITLE
Self-hosted Runner

### DISCUSSION THREAD

**Discussion by ghost** [community/community#26259] [Actions]
<p>We’re using self-hosted runners in our organization and it’s working fine. What I want to do now is use  multiple runners so is there any limit ??</p>
<p>Suppose i have 100 machines and want to create 100 self hosted runners within same organization</p>

**Comment by naikrovek** [comment, marked as answer]
<p>go crazy.  there is no feasible limit.  you will run out of steam before GitHub does.</p>
